packages/babel-plugin-transform-classes/test/fixtures/spec/derived-constructor-must-call-super-5/exec.js

```diff
@@ -0,0 +1,10 @@
+class Bar {}
+
```

Comment:
I would name the folder containing the new test cases `derived-constructor-no-primitive-return` instead of `derived-constructor-must-call-super-5`